cmd/clusterctl/client/config_test.go

```diff
@@ -57,6 +57,7 @@ func Test_clusterctlClient_GetProvidersConfig(t *testing.T) {
 			wantProviders: []string{
 				config.ClusterAPIProviderName,
 				config.KubeadmBootstrapProviderName,
+				config.KubeKeyK3sBootstrapProviderName,
 				config.TalosBootstrapProviderName,
 				config.KubeadmControlPlaneProviderName,
```

Comment:
Probably also needs the KubeKeyK3sBootstrapProviderName

Reply:
done